### PR TITLE
chore: Simplify overrides handler

### PR DESCRIPTION
**Pull request by skhalash**
## Description

Changes proposed in this pull request (what was done and why):

- Simplify overrides.Handler instantiation by decreasing the number of parameters passed into the constructor
- Makde default atomilLevel a singleton defined in the overrides pkg
- Sort constants in main.go

Changes refer to particular issues, PRs or documents:

- 

## Traceability
- [ ] The PR is linked to a GitHub issue.
- [ ] The follow-up issues (if any) are linked in the `Related Issues` section.
- [ ] If the change is user-facing, the documentation has been adjusted.
- [x] The feature is unit-tested.
- [x] The feature is e2e-tested.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -131,21 +129,14 @@ var (
...
-
-	metricOTLPServiceName = "telemetry-otlp-metrics"
-
-	traceOTLPServiceName = "telemetry-otlp-traces"
-
-	selfMonitorName = "telemetry-self-monitor"
+	metricOTLPServiceName         = "telemetry-otlp-metrics"
+	selfMonitorName               = "telemetry-self-monitor"
+	traceOTLPServiceName          = "telemetry-otlp-traces"
+	webhookServiceName            = "telemetry-manager-webhook"
```
Comment (k15r) [inline main.go:139]: nit: lets group them logically
images, services, rest

File: internal/overrides/handler.go [hunk 2]
```diff
@@ -21,17 +34,37 @@ type Handler struct {
...
+type Option = func(*Handler)
+
+func WithAtomicLevel(level zap.AtomicLevel) Option {
+	return func(h *Handler) {
+		h.atomicLevel = level
+		h.defaultLevel = level.Level()
+	}
+}
+
+func DefaultAtomicLevel() zap.AtomicLevel {
```
[the remark this answers is not in the document]
Comment (skhalash) [reply internal/overrides/handler.go]: Yes, this is also how it worked before.

**Review by k15r — APPROVED** [review]

**Pull request merged**